src/api/app/models/package.rb

```diff
@@ -928,8 +928,8 @@ def buildresult(prj = project, show_all = false)
     LocalBuildResult::ForPackage.new(package: self, project: prj, show_all: show_all)
   end
 
-  def jobhistory_list(project, repository, arch, options = {})
-    results = Jobhistory.find_hashed(project: project.name, package: options[:package],
+  def jobhistory_list(project, repository, arch, package_name)
+    results = Jobhistory.find_hashed(project: project.name, package: package_name,
```

Comment:
Please squash this commit into the previous one.

Reply:
why?

Reply:
Because the previous commit is just an interim change that is obsoleted by this commit.